app/frontend/src/main.js

```diff
@@ -92,7 +92,9 @@ function loadKeycloak(config) {
       realm: config.keycloak.realm,
       url: config.keycloak.serverUrl
     },
-    onReady: () => {},
+    onReady: () => {
+      initializeApp(true);
+    },
     onInitError: error => {
       console.error('Keycloak failed to initialize'); // eslint-disable-line no-console
```

Comment:
this should probably raise an error or intializeApp(false).  we only do something on the happy path and nothing on error.

Reply:
I've tested the case where config is loaded, but it has incorrect Keycloak data in order to trigger the onInitError. As this is an asynchronous execution, we don't particularly have to re-execute initializeApp again because the loadConfig function already does this for us. When I added initializeApp to onInitError, it actually caused Vue to completely reload twice, which is unnecessary.